src/availability.rs

```diff
@@ -325,6 +325,44 @@ where
                     .unwrap())
             }
             .boxed()
+        })?
+        .get("get_block_summary", move |req, state| {
+            async move {
+                let id: usize = req.integer_param("height")?;
+
+                state
+                    .get_block(BlockId::Number(id))
```

Comment:
Nit: should be able to just use `id` on its own, as `get_block` is already generic over `T: Into<BlockId>`